src/main/java/com/tcg/lwjgllearning/graphics/g3d/mesh/AttributeColorMesh.java

```diff
@@ -67,9 +57,17 @@ private void createColorBuffer() {
         }
     }
 
+    public void activateAttribute(Attribute attribute, int attributeLocation) {
+        this.activateAttribute4f(attribute, attributeLocation);
+    }
+
     @Override
     public void dispose() {
         glDeleteBuffers(this.colorVboId);
         super.dispose();
     }
+
+    public static enum Attribute implements AbstractAttribute {
```

Comment:
Declaring an inner enum as static is redundant

Reply:
Oh, good to know, didn't realize! Will remove the redundancy.